### DISCUSSION TITLE
Welcome to Franklin!

### DISCUSSION THREAD

**Discussion by inxilpro** [InterNACHI/franklin#3] [General]
Thanks for checking out Franklin! Right now we're really looking for community feedback. Feel free to post questions/comments/ideas.

Also, if you've tested Franklin in your country, please let us know so that we can added it to [the list of tested countries](https://github.com/InterNACHI/franklin/blob/main/docs/src/tested-countries.json) (or feel free to PR your country to that list!)

**Comment by jokorone** [comment]
Input for DE / Germany works as expected 👍

**Comment by kohchihao** [comment]
Input for SG / Singapore works as expected. However, the country is auto selected as Malaysia first, even though I am within Singapore. @inxilpro
